apps/api/src/server/repositories/case.repository.js

```diff
@@ -232,6 +234,7 @@ const updateApplicationSansRegionsRemoval = ({
 	return databaseConnector.case.update({
 		where: { id: caseId },
 		data: {
+			publishedAt: publishedAt || null,
```

Comment:
As we are adding a new piece for data to associate with a case, I don't think there is a matching migration for it

Reply:
I was wondering if that was the case as well, Mayya